templates/puppet.conf

```diff
@@ -1,14 +1,13 @@
+# {{ ansible_managed }}
 [main]
-    logdir = /var/log/puppet
-    rundir = /var/run/puppet
-    ssldir = $vardir/ssl
+    server_urls = https://{{ ansible_fqdn }}:8081
```

Comment:
`ansible_inventory` is probably preferred, probably same is valid for the rest of the `ansible_fqdn` s in the file.

Reply:
Good points. Removed the fqdn and changed to the inventory based variable.